transport/server/play-akka-http-server/src/main/resources/reference.conf

```
@@ -14,6 +14,12 @@ play {
       # How long a request takes until it times out. Set to null or "infinite" to disable the timeout.
       requestTimeout = infinite
 
+      # Timeout after which all requests and connections shall be forcefully terminated
+      # when shutting down the server. It will default to Coordinated Shutdown service-unbind
+      # phase timeout. Value must be a duration, for example:
+      # play.server.akka.terminationTimeout = 10 seconds
+      terminationTimeout = null
```

Nitpick. I'd document that: `Set to null to use de default provided by Coordinated Shutdown service-unbind`.

---

Also, I'd replace the line:

```suggestion
      # terminationTimeout = 10 seconds
```

As in: instead of defaulting to `null` default to not creating the setting (and handle that case on the code). I prefere this approach in isolation but considering the semantics of `requestTimeout = infinite` immediately above I think it'll be more consistent to stay with `terminationTimeout = null`

---

> As in: instead of defaulting to `null` default to not creating the setting (and handle that case on the code)

Thing is that it is better for us to always define the configuration, even if we set it to null. That way we can avoid `ConfigException.Missing` entirely. See https://github.com/lightbend/config#how-to-handle-defaults.

The only difference here is that handling the default is better in code since Akka handles it in code as well.